script.js

```diff
@@ -1,8 +1,90 @@
+/* eslint-disable max-classes-per-file */
+
+let books = JSON.parse(localStorage.getItem('books')) || [];
+
+// |||book class
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+// |||user interface interactions
+class UserInterface {
+  static container = document.getElementById('container');
+
+  static titleInput = document.querySelector('#title');
+
+  static authorInput = document.querySelector('#author');
+
+  static addButton = document.querySelector('.add-book');
+
+  static addBook() {
+    const book = new Book(
+      UserInterface.titleInput.value,
+      UserInterface.authorInput.value,
+    );
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+
+    UserInterface.titleInput.value = '';
+    UserInterface.authorInput.value = '';
+    UserInterface.displayBook(book, books.length - 1);
+  }
+
+  static removeBook(book, index) {
+    const bookContainer = document.getElementById(index);
+    books = books.filter((el) => el !== book);
+    localStorage.setItem('books', JSON.stringify(books));
+    UserInterface.container.removeChild(bookContainer);
+  }
+
+  static displayBook(book, index) {
+    const bookContainer = document.createElement('tr');
+    bookContainer.id = index;
+    const text = document.createElement('td');
+    text.innerHTML = `${book.title} by ${book.author}`;
+
+    bookContainer.append(text);
+    const removeButtonContainer = document.createElement('td');
+
+    const removeButton = document.createElement('button');
+    removeButton.classList.add('button-table');
+    removeButton.innerHTML = "<i class='fas fa-trash-alt'></i> Remove";
+
+    removeButton.onclick = () => {
+      UserInterface.removeBook(book, index);
+    };
+    removeButtonContainer.append(removeButton);
+    bookContainer.append(removeButtonContainer);
+
+    return bookContainer;
+  }
+}
+
+// |||Events -----------------------------------------
+
+// populate the local storage items
+
+books.forEach((book, index) => {
+  const bookContainer = UserInterface.displayBook(book, index);
+  UserInterface.container.appendChild(bookContainer);
+});
+
+// add ba book when button is clicked
+UserInterface.addButton.addEventListener(
+  'click',
+  UserInterface.addBook,
+);
+
+// |||Contact form validation
+
+/*
```

Comment:
- [ ] From this line to the end of the file, you have commented code. Kindly remove commented code as they make your codebase look messy.

  ![abdu-comment](https://user-images.githubusercontent.com/36542986/153035579-bcdae9cd-2ed9-40ec-8090-cb4393ffd4d1.jpg)